src/hono.ts

```diff
@@ -47,24 +63,38 @@ export class Hono {
   }
 
   /* HTTP METHODS */
```

Comment:
Also, please write the features on [`all`](https://github.com/yusukebe/hono/pull/102/files#diff-fef0c99deaa99c63dde9a9af2451ae5281a68e92c338218d644a08b358878245R119)

![SS](https://user-images.githubusercontent.com/10682/153809651-e1e07edb-92f2-4edd-868f-4090a52f8cfa.png)

Reply:
Oops, also added to "all” by 2d2726a